cmd/shell.go

```diff
@@ -159,3 +166,38 @@ func RunUntilSignaled(logger *blog.AuditLogger, server *rpc.AmqpRPCServer, close
 	time.Sleep(time.Second * 5)
 	logger.Warning("Reconnecting to AMQP...")
 }
+
+func ProfileCmd(profileName string, stats statsd.Statter, logger *blog.AuditLogger) {
+	for {
+		var memoryStats runtime.MemStats
+		runtime.ReadMemStats(&memoryStats)
+
+		if err := stats.Gauge(fmt.Sprintf("Gostats.%s.Goroutines", profileName), int64(runtime.NumGoroutine()), 1.0); err != nil {
+			logger.Warning(fmt.Sprintf("Could not send stats to Statsd server: %s", err))
```

Comment:
I generally like this method, but I'd like it more if, when Statsd failed (which shouldn't except for library failure, it's UDP), it didn't print 10 logs / second forever.

I think this is an example of too much error checking. Perhaps we could compromise by logging at Warning once and exiting the goroutine?